github/github.go

```diff
@@ -52,6 +52,11 @@ const (
 	// https://developer.github.com/changes/2014-12-09-new-attributes-for-stars-api/
 	mediaTypeStarringPreview = "application/vnd.github.v3.star+json"
 
+	//https://developer.github.com/v3/media/
```

Comment:
To follow Go style and for consistency, there should be a single space at the beginning of this comment.

See https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes for full rationale.